cloudigrade/config/urls.py

```diff
@@ -4,10 +4,11 @@
 from django.urls import path, include
 from rest_framework import routers
 
-from account.views import AccountViewSet
+from account.views import AccountViewSet, ReportViewSet
 
 router = routers.DefaultRouter()
 router.register(r'account', AccountViewSet)
+router.register(r'report', ReportViewSet, base_name='report')
```

Comment:
Is there a reason for setting the `base_name` here? It doesn't look like we defined a custom `get_queryset` method in the `ReportViewSet `.

Reply:
The `base_name` is necessary because `ReportViewSet` doesn't have a custom `get_queryset`, and I didn't give it a custom `get_queryset` because I couldn't think of a meaningful queryset to put there since all the fun model work for reporting happens a few steps away in `get_hourly_usage`.

Reply:
![](https://i.imgur.com/dVDJiez.gif)